components/skus/common/BUILD.gn

```diff
@@ -11,10 +11,34 @@ static_library("common") {
     "features.h",
   ]
   deps = [ "//base" ]
+
+  if (!is_ios) {
```

Comment:
why is skus_util in common?

Reply:
this depends on third_party/blink/public/platform so it must go in renderer

Reply:
fixed